LICENSE

```diff
@@ -1,13 +1,23 @@
-Copyright (c) 2017 Uber Technologies, Inc.
+The MIT License
```

Comment:
Let's keep the same Apache license. The relicensing should be done more carefully and only if needed.